### PR TITLE
Feature/dashcol 16 create table race control data

### PR DESCRIPTION
**Pull request by FJanza**
Create component TableRaceControls data

### CODE REVIEW

File: src/components/TableRaceControls/components/Flag/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import {FlagProps} from "./types";
+const flags = {
```
Comment (LeandroLescano) [inline src/components/TableRaceControls/components/Flag/index.tsx]:
```suggestion
import React from "react";

import {FlagProps} from "./types";

const flags = {
```

File: src/components/TableRaceControls/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, {useState} from "react";
+import {TableRaceControlsProps} from "./types";
+import moment from "moment";
+import Flag from "./components/Flag";
+import clsx from "clsx";
```
Comment (LeandroLescano) [inline src/components/TableRaceControls/index.tsx]:
```suggestion
import React, {useState} from "react";
import moment from "moment";
import clsx from "clsx";

import Flag from "./components/Flag";
import {TableRaceControlsProps} from "./types";
```

Comment (FJanza) [reply src/components/TableRaceControls/index.tsx]: done

File: src/components/TableRaceControls/components/Flag/index.tsx (hunk 1 above)
Comment (FJanza) [reply src/components/TableRaceControls/components/Flag/index.tsx]: done

**Review by LeandroLescano — DISMISSED** [review]

**Review by LeandroLescano — APPROVED** [review]

**Pull request merged**